### PR TITLE
Fix names in CODEOWNERS

### PR DESCRIPTION
**Pull request by aiuto**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+cpp-tutorial @oquenchil
 java-maven/ @jin
-rules/starlark-configurations @gregce @configurability
-make-variables @gregce @configurability
-AUTHORS  @sventiffe @philwo
+java-tutorial/ @jin
+make-variables @configurability
+flags-parsing-tutorial @configurability
+rules/starlark-configurations @gregestren @configurability
+third-party-dependencies @metorcloudy
```
Comment (gregestren) [inline CODEOWNERS:11]: name typo on this line

File: CODEOWNERS [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
 android/ @ahumesky @jin
+cpp-tutorial @oquenchil
 java-maven/ @jin
-rules/starlark-configurations @gregce @configurability
-make-variables @gregce @configurability
-AUTHORS  @sventiffe @philwo
+java-tutorial/ @jin
+make-variables @configurability
+flags-parsing-tutorial @configurability
+rules/starlark-configurations @gregestren @configurability
```
Comment (gregestren) [inline CODEOWNERS:10]: Should probably just keep this one @configurability like above?

File: CODEOWNERS (hunk 1 above)
Comment (aiuto) [reply CODEOWNERS:11]: fixed

File: CODEOWNERS (hunk 2 above)
Comment (aiuto) [reply CODEOWNERS]: Done

**Review by gregestren — APPROVED** [review]

**Pull request merged**